authors/apps/authentication/views.py

```diff
@@ -167,12 +167,12 @@ def post(self, request):
         user = User.objects.filter(email=user_email).first()
         if user:
             token = user._generate_jwt_token()
-            url = get_current_site(request).domain + reverse('password-change',kwargs={'token':token})
+            url = 'https://authors-haven-dev.herokuapp.com/reset/'
```

Comment:
@dorothyas set this as an environment variable

Reply:
noted